xapian-core/include/xapian/cluster.h

```diff
@@ -33,6 +34,7 @@
 #include <xapian/types.h>
 #include <xapian/visibility.h>
 
+#include <set>
```

Comment:
We don't seem to use `std::set<>` in this header - better to include this from the files which actually use it instead (especially true given this is a public API header).

Reply:
Thanks for pointing this out. Change pushed in a fixup commit.